.gitignore

```diff
@@ -0,0 +1 @@
+.DS_Store
```

Comment:
Having a newline here is important to the upmost

Reply:
i'm gonna kill you